File: deaf_grandma.js
function deafGrandma(){	
  let byeCount = 0;
  let response;
  while (byeCount < 2){
    response = window.prompt('What would you like to say to Grandma?');
    if (response === ''){	
      window.alert('WHAT?!');
    } else if (response === 'GOODBYE!' && byeCount < 1){	
      window.alert('LEAVING SO SOON?');	
      byeCount ++;
    } else if (isUpperCase(response) && response !== 'GOODBYE!'){	
      window.alert('NO, NOT SINCE 1946!');
    } else if (!isUpperCase(response)){	
      window.alert('SPEAK UP, KID!');
    } else if (response === 'GOODBYE!'){	
      window.alert('LATER, SKATER!');	
      return '';
    }
  }
}

function isUpperCase(string){	
  return (string === string.toUpperCase());	
}

deafGrandma();

File: deaf_grandma.py
def deaf_grandma():
  byeCount = 0
  response = ''
  while byeCount < 2:
      response = input('What do you want to say to Grandma??')
      if response == '':
        print('WHAT?')
      elif response == 'GOODBYE!' and byeCount < 1:
        print('Leaving so soon?')
        byeCount +=1
      elif response.isupper() and response != 'GOODBYE!':
        print('NO NOT SINCE 1946!')
      elif not response.isupper():
        print('SPEAK UP KID!')
      else:
        print('LATER SKATER!')
        byeCount +=1


deaf_grandma()
